.github/scripts/test_dynamically.py

```diff
@@ -6,10 +6,14 @@
 import typer
 from bioimageio.spec import load_raw_resource_description
 from bioimageio.spec.shared import yaml
-from tqdm import tqdm
 from update_log import add_log_entry
 
-tqdm.__init__ = partialmethod(tqdm.__init__, disable=True)  # silence tqdm
+try:
+    from tqdm import tqdm
+except ImportError:
+    pass
+else:
+    tqdm.__init__ = partialmethod(tqdm.__init__, disable=True)  # silence tqdm
```

Comment:
I might be misreading but this should fail if no tqdm, right?

Reply:
no on ImportError there is a `pass` the `else` runs if there was no exception. So this should be fine

Reply:
Oh right... I guess as it's only a one line "else" section would have probably just included it in the "try" but your way is more correct I expect